airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/message/MessageQueue.kt

```diff
@@ -76,6 +76,8 @@ class DefaultQueueChannel(override val messageQueue: MessageQueue<*, Destination
 }
 
 @Singleton
+// Why inject this factory? Why not inject DefaultQueueChannel directly instead?
+// This comment also applies elsewhere.
```

Comment:
You're right that this doesn't need to be a factory anymore. It's factored down from something that had injected dependencies.